core/backend/src/ECSqlStatement.ts

```diff
@@ -332,46 +336,40 @@ export class ECSqlStatement implements IterableIterator<any>, IDisposable {
    * - [Code Samples]($docs/learning/backend/ECSQLCodeExamples#working-with-the-query-result)
    */
   public getRow(args?: ECSqlRowArg): any {
-    args = args ?? {};
+    if (!this._stmt)
+      throw new Error("ECSqlStatement is not prepared");
 
-    // Set default rowFormat if not provided
+    args = args ?? {};
     if (args.rowFormat === undefined) {
       args.rowFormat = QueryRowFormat.UseJsPropertyNames;
     }
-
-    const toRowOptions = {
-      classIdsToClassNames: true,
-      rowFormat: args.rowFormat,
-    };
-    const resp = this._stmt!.toRow(toRowOptions); // eslint-disable-line @typescript-eslint/no-non-null-assertion
+    const resp = this._stmt.toRow({
+      classIdsToClassNames: args.classIdsToClassNames,
+      useJsName: args.rowFormat === QueryRowFormat.UseJsPropertyNames,
+      abbreviateBlobs: false,
+      // on 5.x this should be removed or set to false. It is only here to maintain compatibility with 4.x
```

Comment:
```suggestion
      // on 5.x this should be removed or set to false. It is only here to maintain compatibility with 4.x
      In 4.x, people are currently dependent on the behavior of aliased classIds `select classId as aliasedClassId` not being converted into classNames which is a bug that we must now support. This option preserves this special behavior until it can be removed in a future version.
```

Probably could be better, but I would like to better capture why this option is necessary